code/src/components/ClearAll.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import { todos } from 'reducers/todos'
+
```

Comment:
Dont forget to be consistent with the separation of dependencies and local imports

import React from 'react'
import { useDispatch } from 'react-redux'
import styled from 'styled-components'

import { todos } from 'reducers/todos'